golos.publication/golos.publication.cpp

```diff
@@ -787,4 +789,30 @@ void publication::reblog(name rebloger, name author, std::string permlink) {
             "You can't reblog, because this message doesn't exist.");
 }
 
+int64_t publication::pay_delegators(uint64_t message_id, int64_t claim, name voter, eosio::symbol tokensymbol) {
+    int64_t dlg_payout_sum = 0;
+    auto delegate_list = golos::vesting::get_list_delegate(config::vesting_name, voter, tokensymbol.code());
```

Comment:
you may not use this list, it can be very different from list of delegators at a time of upvote. use delegators list from vote_table (weight and strategy should be there too)